docsbox/docs/views.py

```diff
@@ -170,42 +169,21 @@ def get(self, task_id):
         return response
 
 
-def get_file_info(req, file_id):
+def get_file_info(file_id, filename="", mimetype=None):
     file_info = {}
-    mimetype = req.headers.get('Content-Type')
-    if mimetype is None or mimetype == "application/pdf" or mimetype not in app.config["CONVERTABLE_MIMETYPES"]:
-        if db.exists('fileId:' + file_id) == 0:
-            filename, tmp_file_path, mimetype = via_comunication(file_id)
-            file_info = {"file_path": tmp_file_path, "mimetype": mimetype, "filename": filename, "datetime": datetime.now().strftime('%Y/%m/%d-%H:%M:%S')}
-            db.set('fileId:' + file_id, json.dumps(file_info))
-        else:
-            file_info = json.loads(db.get('fileId:' + file_id))
-            if not check_file_path(file_info["file_path"]):
-                filename, tmp_file_path, mimetype = via_comunication(file_id)
-                file_info = {"file_path": tmp_file_path, "mimetype": mimetype, "filename": filename, "datetime": datetime.now().strftime('%Y/%m/%d-%H:%M:%S')}
-                db.set('fileId:' + file_id, json.dumps(file_info))
-            elif file_info["filename"] == "" and req.headers.get('Content-Disposition'):
-                file_info["filename"] = req.headers.get('Content-Disposition')
-                file_info["datetime"] = datetime.now().strftime('%Y/%m/%d-%H:%M:%S')
-                db.set('fileId:' + file_id, json.dumps(file_info))
+    if db.exists('fileId:' + file_id) == 0:
+        if mimetype is None or mimetype == "application/pdf" or mimetype not in app.config["CONVERTABLE_MIMETYPES"]:
```

Comment:
Why are we re-checking the mimetype if it is application/pdf or not in convertable types?